codebuild_specs/e2e_workflow.yml

```diff
@@ -755,61 +755,62 @@ batch:
           CLI_REGION: ca-central-1
       depend-on:
         - publish_to_local_registry
-    - identifier: utils_ddb_iam_access_data_construct_custom_logic_amplify_table_5
+    - identifier: >-
+        ddb_iam_access_data_construct_custom_type_iam_access_custom_logic_amplify_table_5
       buildspec: codebuild_specs/run_cdk_tests.yml
       env:
         compute-type: BUILD_GENERAL1_LARGE
         variables:
           NODE_OPTIONS: '--max-old-space-size=14848'
           TEST_SUITE: >-
-            src/__tests__/utils.test.ts|src/__tests__/ddb-iam-access.test.ts|src/__tests__/data-construct.test.ts|src/__tests__/custom-logic.test.ts|src/__tests__/amplify-table-5.test.ts
+            src/__tests__/ddb-iam-access.test.ts|src/__tests__/data-construct.test.ts|src/__tests__/custom-type-iam-access.test.ts|src/__tests__/custom-logic.test.ts|src/__tests__/amplify-table-5.test.ts
```

Comment:
The new test creates 2 lightweight stacks (new API, NONE data source, IAM role; no user pool, DDB or SQL data sources), so it should be safe to bundle with these other tests. If we start encountering issues with this step, we can split the new test into its own bundle.